thrift/compiler/generate/t_php_generator.cc

```diff
@@ -1320,14 +1326,90 @@ void t_php_generator::_generate_php_struct_definition(
       << tstruct->get_name() << param << endl
       << indent() << "}" << endl
       << endl;
-
+  
+  generate_generic_field_getters_setters(out, tstruct); 
   generate_php_struct_reader(out, tstruct);
   generate_php_struct_writer(out, tstruct);
   generate_json_reader(out, tstruct);
   indent_down();
   out << indent() << "}" << endl << endl;
 }
+/**
+ * Generates necessary accessors and mutators for the private fields
+ */
+void t_php_generator::generate_generic_field_getters_setters(std::ostream& out,
+                                                              t_struct* tstruct) {
+  std::ostringstream getter_stream;
+  std::ostringstream setter_stream;
+
+  // build up the bodies of both the getter and setter at once
+  const vector<t_field*>& fields = tstruct->get_members();
+  vector<t_field*>::const_iterator f_iter;
+  for (f_iter = fields.begin(); f_iter != fields.end(); ++f_iter) {
```

Comment:
nit: This can be simplified by
`for (const auto& field : tstruct->get_members()) { ... }`

Reply:
Done!

Reply:
Done! Although since I am using the Github web version, I missed out on the brace and can't squash the commits from my end. If it has to be merged, could you squash it from your end?